src/main/java/org/alfresco/repo/copy/CopyServiceImpl.java

```diff
@@ -1519,4 +1520,22 @@ public String getTopLevelNodeNewName(NodeRef sourceNodeRef, NodeRef targetParent
         return null;
     }
 
+    /**
+     * Builds name by appending copy label.
+     */
+    String buildNewName(NodeRef destinationParent, QName assocTypeQName, String newName)

```

Comment:
You should remove the redundant parameters destinationParent and assocTypeQName. Same as comment on CopyServiceImpl.java